app/views/layouts/homepage.html.erb

```diff
@@ -0,0 +1 @@
+<%= render template: 'layouts/sufia' %>
```

Comment:
oops, actually, not sure this will work. layouts/sufia has the wrapping `<html><body>` etc tags... which need to be on the outside, not wrapped by other content. 

May need to copy the layouts/sufia locally. :(  Ideally, the layouts/sufia might use partial(s) that we can reuse... but it doesn't. We could PR that to hyrax.